specfile/README.md

```diff
@@ -0,0 +1,75 @@
+# Packit requirements for the spec-file library
+
+## Manipulation
+
+- We create a Python object by specifying a path of the spec-file.
+- (optional) After a change, we can reload the content (can be done by recreating of the object).
+- After the manipulation, the content is synced to the file or, we have a way to do this explicitely.
+- Be able to copy a section of spec-file to another object.
+
+## Version and release
+
+- The version and release fields can be get.
+- We can set the new version. If release is not specified, reset the release number to 1.
```

Comment:
```suggestion
- We can set the new version. If release is not specified, reset the release number to 1 while preserving all the macros.
```

Reply:
What about mentioning this in the beginning that we want to save only relevant changes, not the expanded macros.

Reply:
I've added `All changes results in a minimal and local diff.` to the very beginning.